README.adoc

```diff
@@ -189,18 +192,15 @@ method and integrate it into the [hotspot=PropertiesForHost file=0]`getPropertie
 ----
 
 The contents of your inventory are now under the asterisk (*) wildcard and reside at the following URL:
+http://localhost:9080/inventory/hosts/*[http://localhost:9080/inventory/hosts/*^]
```

Comment:
- [ ] Format of URL in "Linking to inventory contents" section is all weird: 
![image](https://user-images.githubusercontent.com/46586499/73955691-bfb69480-48d1-11ea-9780-f0fdef39c4c5.png)